easybuild/easyblocks/m/mumps.py

```diff
@@ -34,12 +34,13 @@
 
 import os
 import shutil
+from distutils.version import LooseVersion
 
 import easybuild.tools.toolchain as toolchain
 from easybuild.easyblocks.generic.configuremake import ConfigureMake
 from easybuild.tools import toolchain
 from easybuild.tools.build_log import EasyBuildError
-from easybuild.tools.modules import get_software_root
+from easybuild.tools.modules import get_software_root, get_software_version
```

Comment:
please merge https://github.com/jordiblasco/easybuild-easyblocks/pull/1 to fix this